modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -4,12 +4,14 @@
 // commit: 7ad14e7a3e6f87ddc450f7e34621eb5200808451
 // file: core/variant_call.cpp
 // commit: 5ad9be4c24e9d7dc5672fdc42cea896622fe5685
+
```

Comment:
Maybe it's time we just remove these comments at the top of these files?

Reply:
Yes, I forgot these were still there.